crates/rattler_repodata_gateway/src/fetch/mod.rs

```diff
@@ -722,13 +722,17 @@ fn validate_cached_state(cache_path: &Path, subdir_url: &Url) -> ValidatedCacheS
                 max_age: Some(duration),
                 ..
             }) => {
-                if duration > cache_age {
-                    tracing::info!("Cache is out of date. Assuming out of date...");
+                if cache_age > duration {
+                    tracing::debug!(
```

Comment:
Is `debug` something that is not compiled into the final binary? I think information about the cache is kinda informative to analyze wether the cache is hit or not, so I'd be OK with keeping it at `info` (if debug is gone at runtime) so that users could access this information by changing the verbosity flag.

Reply:
It is my understanding that all these statements are left in. Let me verify.

Reply:
Yep they are :)